bundles/admin/sulu-buttons.rst

```diff
@@ -0,0 +1,131 @@
+Sulu-Buttons
+============
+
+The husky-toolbar-component needs to be passed data to which defines what buttons it should render or what happens
+when somebody clicks on a button. Essentially Sulu-buttons are simply about how do I pass buttons to the
+toolbar-component more easily and elegantly than just copying the same buttons all over the place.
```

Comment:
Hard to read... what about: `Essentially Sulu-Buttons are about passing buttons to the toolbar-component in an easier and more elegant way than ...`?